jetson/percep/pcl.cpp

```diff
@@ -480,4 +481,4 @@ void PCL::update() {
     std::cerr << "Height: "<< pt_cloud_ptr->height<<"\n";
 }
 
-#endif
+#endif
```

Comment:
add back the newline please (https://stackoverflow.com/questions/72271/no-newline-at-end-of-file-compiler-warning)